packages/module/patternfly-docs/generated/index.js

```diff
@@ -5,15 +5,19 @@ module.exports = {
     toc: [[{"text":"With complex toolbar"}]],
     examples: ["With complex toolbar"],
     section: "extensions",
+    subsection: "",
     source: "react-demos",
+    tabName: null,
     Component: () => import(/* webpackChunkName: "extensions/log-viewer/react-demos/index" */ './extensions/log-viewer/react-demos')
   },
   '/extensions/log-viewer/design-guidelines': {
     id: "Log viewer",
     title: "Log viewer",
     toc: [{"text":"Light theme log viewer"},{"text":"Dark theme log viewer"},{"text":"Usability"},{"text":"Log viewer functionality"},[{"text":"With popover"},{"text":"With dropdown, drilldown, and search expanded"},{"text":"With search results"}]],
     section: "extensions",
+    subsection: "",
     source: "design-guidelines",
+    tabName: null,
```

Comment:
should we use undefined instead?

Reply:
Hmm, I think that tabName value is just coming from the docs framework, and it's coming back as null because we don't define a specific one. It's just showing up as a change here because I bumped the docs framework version.

But yeah that seems like it might be a change we want to make in the docs framework.

Reply:
Yeah that frontmatter value is coming from this PR in org https://github.com/patternfly/patternfly-org/pull/3394/files